### PR TITLE
global: add copyright everywhere

### PR DESCRIPTION
**Pull request by jalavik**
- Adds copyright information to all files.
- Updates setup.py.

@eamonnmag and @jstypka can you go over this and let me know if it is okay. I added codeauthor to every Python file listing who has contributed to it.

### CODE REVIEW

File: inspire_magpie/static/js/utils.js [hunk 1]
```diff
@@ -1,4 +1,11 @@
 /**
+ * This file is part of Inspire-Magpie.
+ * Copyright (c) 2016 CERN
+ *
+ * Inspire-Magpie is a free software; you can redistribute it and/or modify it
+ * under the terms of the MIT License; see LICENSE file for
+ * more details.
+ *
  * Created by eamonnmaguire on 23/02/2016.
```
Comment (jstypka) [inline inspire_magpie/static/js/utils.js]: do we keep also the `created by` comments?

Comment (jalavik) [inline inspire_magpie/static/js/utils.js]: Ah, right. Shall we remove this from static files? @eamonnmag

File: inspire_magpie/ui.py [hunk 2]
```diff
@@ -27,7 +45,7 @@ def extractor():
 
 @blueprint.route('/extract-feedback', methods=['POST'])
 def extract_feedback():
-    print request.form
+    print(request.form)
```
Comment (jstypka) [inline inspire_magpie/ui.py]: isn't it debugging code? We might want to remove it altogether

**Comment by jstypka** [comment]
Looks good, we should just check that installing it from scratch works works with the new `setup.py`

File: inspire_magpie/static/js/utils.js (hunk 1 above)
Comment (eamonnmag) [inline inspire_magpie/static/js/utils.js]: Yeah, it's put there automatically by pycharm :)

**Comment by jalavik** [comment]
@jstypka @eamonnmag Thanks! So, I have checked that it installs nicely and this version is now running on http://inspire-magpie-web1.cern.ch/ as well.

**Comment by eamonnmag** [comment]
Only thing now is - if you use word2vec with just positive words and no negative, there is a server error.

**Comment by jalavik** [comment]
Issue created #5 Gonna merge this then.

**Pull request merged**